packages/projects-docs/pages/learn/editors/_meta.json

```diff
@@ -0,0 +1,6 @@
+{
+  "overview": "Overview",
+  "web": "Web",
+  "ios": "iOS",
+  "vscode": "VSCode"
```

Comment:
This should be `"vs-code": "VS Code"`.
I tried changing it directly but it broke things and I couldn't figure out why.

Reply:
hmm, I changed the text and it seemed to work 🤷